npm/private/npm_link_package.bzl

```diff
@@ -347,8 +347,9 @@ def _impl_direct(ctx):
         DefaultInfo(
             files = depset(output_files, transitive = [ctx.attr.src[DefaultInfo].files]),
         ),
-        declaration_info(
-            declarations = depset(output_files, transitive = [ctx.attr.src[DeclarationInfo].transitive_declarations]),
+        DeclarationInfo(
```

Comment:
👍 for just directly using `DeclarationInfo` unless the factory method has actual logic in it, imo...

Reply:
I'm actually thinking of removing all our DeclarationInfo uses through the rule sets and carrying declarations in JsInfo. Working towards that here.

Reply:
I'd want to think about that, fine grained vs all-in-one 🤔

Doesn't that mean if all you need is the `.d.ts` it might force other files to be generated? But having tons of tiny providers is a hassle and has its own problems of course.

Reply:
This is out of date now that the code has been refactored further